gufe/tests/test_alchemicalnetwork.py

```diff
@@ -13,8 +13,8 @@
 class TestAlchemicalNetwork(GufeTokenizableTestsMixin):
 
     cls = AlchemicalNetwork
-    key = "AlchemicalNetwork-8c6df17d7ecf5902e2e338984cc11140"
-    repr = "<AlchemicalNetwork-8c6df17d7ecf5902e2e338984cc11140>"
+    key = "AlchemicalNetwork-d1035e11493ca60ff7bac5171eddfee3"
```

Comment:
Just checking: is this changing the gufe key because you've changed the fixture to use non-`None` settings? (Otherwise, nothing in this PR should affect gufe keys, right?)

Reply:
yep it's this